cla-backend-go/v2/cla_manager/service.go

```diff
@@ -57,6 +57,8 @@ var (
 	ErrNoOrgAdmins = errors.New("no Admins in company ")
 	//ErrRoleScopeConflict thrown if user already has role scope
 	ErrRoleScopeConflict = errors.New("role or scope conflict")
+	//ErrNotOrgAssociated thrown if user is not associated with org
+	ErrNotOrgAssociated = errors.New("user is not associaeted with organization")
```

Comment:
In this PR, below you use a different error message: ErrNotInOrg.  Is this error message used anywhere?